tsconfig.json

```diff
@@ -38,5 +38,5 @@
     "node_modules/jest-styled-components/typings/index.d.ts"
   ],
   "exclude": ["node_modules", "lib"],
-  "include": ["src", "example"]
+  "include": ["src", "example", "tsconfig.es5.json"]
```

Comment:
I don't think the `include` option is meant to include any tsconfig files, it should just be for source files as far as I know.